README.md

```diff
@@ -16,6 +16,8 @@ If you are the repository administrator, first configure `focus` for your repo u
 
 For end-users, see [Usage](focus/doc/usage.md) for instructions on how to use `focus` to manage your sparse checkouts.
 
+For new or curious users, see [Bazel Tutorial](focus/doc/bazel_tutorial.md) for instructions on how to try `focus` on [bazel](https://github.com/bazelbuild/bazel)
```

Comment:
```suggestion
For new or curious users, see [Bazel Tutorial](focus/doc/bazel_tutorial.md) for instructions on how to try `focus` on the [bazel repository](https://github.com/bazelbuild/bazel) itself.
```

It's probably worth specifying that we mean to have them try it out on the repository, not for the build system Bazel (as opposed to e.g. Pants, like how we promised that you can use multiple build systems).